src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot;
+
+public final class Constants {
+  public static final double stickDeadband = 0.1; 
+  public static final class Drive {
+    /* Drive Constants */
+    public static final int wheelDiameter = 4;
+    public static final double neoDriveGearRatio = 6.12;
+    public static final double angleGearRatio = 21.4286;
+    public static final double encoderResolution = 42;
+    public static final double falconDriveGearRatio = 6.75;
```

Comment:
You could do more with ternaries here, to keep configuration isolated to the Constants file. For example, after you set isNeo, you can have just one variable called `driveGearRatio` that changes based on `isNeo`. Doing this for path as well would eliminate configuration logic in Drive.